<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletCreateAccountClass</class>
 <widget class="QWidget" name="AppletCreateAccountClass">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>245</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create New Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>No Registraion Required.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Just Enter A Online User Name and Log in!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_UserNameLabel">
     <property name="text">
      <string>User Name: ( 4 to 23 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_UserNameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="m_MoodMessageLabel">
     <property name="text">
      <string>Mood Message: ( Up to 27 characters )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_MoodMessageEdit">
     <property name="text">
      <string>Let's Communicate!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_AgeEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Preferred Content:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_ContentComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_GenderComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Primary Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_LanguageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_LoginButton">
     <property name="text">
      <string>Login!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
